gtk-subclass/tests/simple_application.rs

```diff
@@ -213,6 +218,6 @@ fn test_create() {
     // this runs the mainloop, so we can't use gtk-test here!
     application.run(&["--local".to_string()]);
 
-    assert!(unsafe{WINDOW_WAS_VISIBLE})
+    assert!(!*application.window_was_visible.borrow())
```

Comment:
You want to assert that this is true, or not?

Reply:
Erh, I forgot to save the file after I rectified. That was to test if the thing would actually panic if I reversed the condition